### PR TITLE
Tests for JsonHandler

### PR DESCRIPTION
**Pull request by r-sreesaran**
Added Tests for JsonHandler class

### CODE REVIEW

File: openwebstart/src/test/java/com/openwebstart/jvm/json/JsonHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonSyntaxException;
+import  com.openwebstart.jvm.json.Person;
+import com.openwebstart.jvm.runtimes.LocalJavaRuntime;
+import com.openwebstart.jvm.runtimes.RemoteJavaRuntime;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (sclassen) [inline openwebstart/src/test/java/com/openwebstart/jvm/json/JsonHandlerTest.java:11]: No star imports please

File: openwebstart/src/test/java/com/openwebstart/jvm/json/Person.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.openwebstart.jvm.json;
+
+public class Person {
+    String name;
```
Comment (sclassen) [inline openwebstart/src/test/java/com/openwebstart/jvm/json/Person.java:4]: make fields `private`

**Review by sclassen — CHANGES_REQUESTED** [review]
Thank you for this contribution to OpenWebStart !!

Some minor changes are required, mostly formatting.

Also could you please add another test which converts a Person -> Json -> Person and then assert that the age and name are the same as the original.

File: openwebstart/src/test/java/com/openwebstart/jvm/json/JsonHandlerTest.java (hunk 1 above)
Comment (r-sreesaran) [reply openwebstart/src/test/java/com/openwebstart/jvm/json/JsonHandlerTest.java]: done

**Comment by r-sreesaran** [comment]
requested changes were done.

**Comment by hendrikebbers** [comment]
This looks quite good in general. One important point is missing here:

Internally the Json handler is used to deserialise the given classes `LocalJavaRuntime`, `RemoteJavaRuntime`, and `RemoteRuntimeList`. Can you add some tests (deserialisation will be enough) for this classes?

**Comment by r-sreesaran** [comment]
Above mentioned scenarios were covered.

**Comment by r-sreesaran** [comment]
Anything else pending here ?

**Comment by sclassen** [comment]
Sorry to have you hanging here.
We are currently looking with legal department if we need a CLA for contributions. Will give you an update on Monday.

**Comment by sclassen** [comment]
@r-sreesaran
Can you please go to https://dev.karakun.com/cla/ and fill out the appropriate CLA.
Once this is done we can merge this.
Thank you for your patience.

Oh and thank you for being our guinea pig for this process.
You will be the first to do this.

Could you quickly drop a line here once you sent the CLA so we can make sure everything runs smooth.

**Review by sclassen — APPROVED** [review]

**Comment by r-sreesaran** [comment]
CLA sent.

**Comment by sclassen** [comment]
@cla-bot check

**Comment by sclassen** [comment]
Thank you. all worked out fine with the CLA.
Happy to receive more contributions from you  :)

**Pull request merged**